model-optimizer/extensions/front/kaldi/replace_lstm_nonlinearity.py

```diff
@@ -27,12 +27,26 @@ def run_before(self):
         return [FullyConnectedDecomposer]
 
     def replace_op(self, graph: Graph, node: Node):
-        # split input to (i_part, f_part, c_part, o_part, ct_1)
         node_name = node.soft_get('name', node.id)
+        # check if we have dropout
+        input_port = node.in_port(0)
+        if node['use_dropout']:
+            split_dropout = AttributedVariadicSplit(graph,
+                                                    {'name': node_name + '/split_dropout',
+                                                     'size_splits': np.array([-1, 1, 1, 1],),
+                                                     'axis': np.int64(1)}).create_node()
+            input_port.get_connection().set_destination(split_dropout.in_port(0))
+            input_port = split_dropout.out_port(0)
```

Comment:
please create several variable for each port and name them properly. And use below.
`... = split_dropout.out_port(1)` etc.